R/stats-kmeans-tidiers.R

```diff
@@ -62,7 +62,7 @@ augment.kmeans <- function(x, data, ...) {
 
   # kmeans allows for input matrices without column names,
   # so add them in the same way that fix_data_frame() would have
-  if (inherits(data, "matrix") & is.null(colnames(data))) {
+  if (inherits(data, "matrix") && is.null(colnames(data))) {
```

Comment:
Could you pull these changes from `&` to `&&` out into a separate PR, clarifying the bugs you see arising from the current implementation?

Reply:
Opened #1182, #1181. I will update this PR when they are merged.